source/slang/slang-ir-autodiff-transcriber-base.cpp

```diff
@@ -445,8 +457,17 @@ IRType* AutoDiffTranscriberBase::differentiateExtractExistentialType(IRBuilder*
     auto interfaceType = as<IRInterfaceType>(unwrapAttributedType(origType->getOperand(0)->getDataType()));
     if (!interfaceType)
         return nullptr;
-    List<IRInterfaceRequirementEntry*> lookupKeyPath = differentiableTypeConformanceContext.findDifferentiableInterfaceLookupPath(
+    List<IRInterfaceRequirementEntry*> lookupPathValueType = differentiableTypeConformanceContext.findInterfaceLookupPath(
```

Comment:
This might be from before the inheritance flattening patch.. you can have `interfaceType:middleInterface:IDifferentiable` so this was for emitting lookup insts into the current block, to recurse into the sub-type witness, if the witness is a run-time variable.

Post-flattening, is there no need for this?